### ISSUE TITLE
DAC 15 - Medium Priority WCAG Level AA - Colour Contrast [Non-Text] 02

### ISSUE THREAD

**Issue by JadeneAd** [ONSvisual/income-scrolly#15]
<img width="238" alt="DAC_Colour_Contrast_Non_Text_02" src="https://user-images.githubusercontent.com/75850106/155558178-d4caa9a3-0324-473a-b710-db90d970a0e3.png">

### Issue
Low vision users that mouse over any of the interactive elements within the cluster of places underneath the heading ‘Income deprivation in Adur’ will find that the orange colour used may not be clearly visible. This is because the colour contrast is less than 3:1 against adjacent colours.
Foreground: #FFA300 Background: #FFFFFF
The contrast ratio is: 2:1 1.4.11 Non-text Contrast (AA)
Fail for UI components and graphical objects

<img width="597" alt="DAC 15 Current code ref" src="https://user-images.githubusercontent.com/75850106/155558267-65a6f92d-93ff-4926-a55c-bed1bff85a6e.png">

**Low vision comments:**
“The orange ring when hovering over the dots in the chart fails non-text colour contrast at an approximate ratio of 1.3:1, this should be at least 3:1.”

### Suggested Solution
Ensure that the colour contrast is 3:1 against adjacent colours.
